.github/workflows/yaml-lint.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: YAML Lint
+
+on:
+  push:
+    paths:
+      - "**.yml"
+      - "**.yaml"
+  pull_request:
+    paths:
+      - "**.yml"
+      - "**.yaml"
+
+jobs:
+  yamllint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+
+      - name: Install yamllint
```

Comment:
Does it make sense to use a one-step yamllint action? I don't have any personal experience with them but here are some options:
https://github.com/marketplace/actions/yaml-lint
https://github.com/marketplace/actions/yamllint-github-action